src/nativeTest/kotlin/commands/ClostTest.kt

```diff
@@ -0,0 +1,64 @@
+package commands
+
+import api.IEclairClientBuilder
+import kotlinx.cli.ArgParser
+import kotlinx.cli.ExperimentalCli
+import kotlinx.serialization.json.Json
+import mocks.DummyEclairClient
+import mocks.DummyResultWriter
+import mocks.FailingEclairClient
+import types.ApiError
+import kotlin.test.*
+
+class CloseCommandTest {
```

Comment:
nit: there is a typo in the file name, it should be `CloseTest.kt`

Reply:
Corrected this in af870ba3533bd59cec3288ffb59f4c91e14546d0